examples/layers/preprocessing/bounding_box/resizing_demo.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
Can we test this with the RetinaNet training script?

Reply:
Yeah; Once we have RandomCropAndResize we'll have a much clearer value add here I think.  Added to the proc eval function portion.

Reply:
Can you clarify? Given this has crop and pad flags, how do we expect OD pipelines to use Resizing and RandomCropAndResize?